src/main/java/utility/DecoderLocal.java

```diff
@@ -0,0 +1,12 @@
+package utility;
+
+import lombok.experimental.UtilityClass;
+
+import static java.util.Base64.*;
```

Comment:
As far as I know, using an asterisk is not the best practice. Could you please replace it with a specific import?